### ISSUE TITLE
Load failed, save is disabled

### ISSUE THREAD

**Issue by szagoruyko** [facebook/iTorch#56]
When running a remote client I cannot load or save anything, except from orange bar saying "load failed, save is disabled" and a get error with custom.css there is no other debug info. Local client is fine.

**Comment by soumith** [comment]
is this because you did not make the notebook "trusted" from the itorch dropdown menu?

**Comment by szagoruyko** [comment]
no, that's the first thing I try when have issues with itorch :)

**Comment by soumith** [comment]
oh sigh! is this with ipython 3.xx or 2.xx?

**Comment by szagoruyko** [comment]
3.2.0

**Comment by szagoruyko** [comment]
can there be a workaround or downgrade? 2.xx was working

**Comment by soumith** [comment]
downgrade is easy.

conda remove ipython
conda install ipython=2.4.2

**Comment by szagoruyko** [comment]
thanks, it works with 2.4.1
